cardano-cli/src/Cardano/CLI/Compatible/Exception.hs

```diff
@@ -69,3 +70,6 @@ fromExceptTCli
   => ExceptT e IO a
   -> m a
 fromExceptTCli = withFrozenCallStack $ fromEitherIOCli . runExceptT
+
+readFileCli :: (HasCallStack, MonadIO m) => FilePath -> m ByteString
+readFileCli = withFrozenCallStack . readFileBinary
```

Comment:
Wrong place to define file functions. If it's meant to be temporary, leave a TODO comment.

Reply:
Where is the "right" place? It's not temporary.

Reply:
`Cardano.CLI.Read` I guess. We should break it apart at some point.